plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -835,6 +837,49 @@ public void testCreateOrReplaceWithTableChangesFunction()
         }
     }
 
+    @Test
+    public void testIcebergTablesFunction()
```

Comment:
PTAL https://trino.io/docs/current/develop/tests.html#conventions-and-recommendations
```
Test classes should be defined as package-private and final
Test methods should be defined as package-private
```

Reply:
This class is an abstract class on purpose and the methods are being overridden in the `TestIcebergUnityRestCatalogConnectorSmokeTest`, including this new test.
I did rework the override in the `BaseIcebergMinioConnectorSmokeTest` (it was not correct anyway).